src/components/ContextMenu/utils.js

```diff
@@ -0,0 +1,127 @@
+import empty from '../../assets/empty.png';
+
+export const contextMenuData = [
+  {
+    type: 'menu',
+    icon: empty,
```

Comment:
Wait, why? Let's say you do support icons in menus (and it looks like you don't) - Make the `icon` optional and just leave it out when not needed. What's the rational here?

Reply:
You're right 👍. I intended to use it, but forgot. The current data structure was something temporary and I forgot about it. Since its way to easy to just remove all the icons, I added another feature. You can pick how to arrange the icons under the "Arrange Icons By" sub-menu. it's not actually doing anything at the moment, but at least it looks more like the real context menu.